### ISSUE TITLE
App login broken over 5G

### ISSUE THREAD

**Issue by FlattusBlastus** [jellyfin/jellyfin-android#1498]
### Describe the bug

1. Configure JF via RP - NPM official JF documentation; works on remote networks by browser
2. Restrict phone to cell networks only
3. Will not login via https://jf.domain.com

### Logs

```shell
n/a
```

### Application version

2.6.0

### Where did you install the app from?

Google Play

### Device information

OnePlus 12

### Android version

Android 14

### Jellyfin server version

10.9.11

### Which video player implementations does this bug apply to?

- [ ] Web player (default)
- [ ] Integrated player (ExoPlayer)
- [ ] External player (VLC, mpv, MX Player)

**Comment by ArceyTheSecond** [comment]
I too had the issue where I could log in externally with the browser, but not the app. I had me running in circles for days!

With me the issue was resolved when I turned off the IPV6 flag for my domain (that is, where my actual domain is registered, with Dynu). 

Turns out that the app was trying to resolve my IP for jellyfin.domain.com through IPV6, where I didn't actually have that available on the IP where it pointed to. So turning that off resolved it for me.
